packages/react-component-library/src/components/RadioE/partials/StyledRadio.tsx

```diff
@@ -10,17 +10,33 @@ interface StyledRadioProps {
   $isChecked?: boolean
 }
 
-const RADIO_FOCUS_WIDTH = '0.1rem'
+const RADIO_ACTIVE_BORDER_WIDTH = '2px'
 
 const { spacing, fontSize, color, animation } = selectors
 
+const BackgroundColour = css<StyledRadioProps>`
```

Comment:
Was getting a bit repetitive and confusing following the `$isDisabled` overrides, so I've used a slightly different approach here.